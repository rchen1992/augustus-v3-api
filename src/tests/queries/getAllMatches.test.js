const mockMatchesRepo = require('@tests/mockRepos/match');
const { testQuery } = require('@tests/utils');

testQuery({
    id: 'Get all matches basic data',
    query: `
        {
            matches {
                id
                tied
                createdAt
                updatedAt
            }
        }
    `,
    variables: {},
    context: () => {
        return {
            repos: {
                match: mockMatchesRepo,
            },
        };
    },
    expected: undefined,
});
