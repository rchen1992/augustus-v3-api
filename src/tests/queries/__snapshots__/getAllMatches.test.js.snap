// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`query: Get all matches basic data 1`] = `
Object {
  "data": Object {
    "matches": Array [
      Object {
        "createdAt": "2018-12-07 05:46:42",
        "id": "1",
        "tied": false,
        "updatedAt": "2018-05-11 06:33:00",
      },
      Object {
        "createdAt": "2018-08-22 00:10:00",
        "id": "2",
        "tied": false,
        "updatedAt": "2019-03-23 04:57:33",
      },
      Object {
        "createdAt": "2018-05-01 17:04:52",
        "id": "3",
        "tied": false,
        "updatedAt": "2018-12-16 02:16:10",
      },
      Object {
        "createdAt": "2018-09-17 01:38:13",
        "id": "4",
        "tied": false,
        "updatedAt": "2018-05-27 00:41:58",
      },
      Object {
        "createdAt": "2019-04-06 07:39:01",
        "id": "5",
        "tied": false,
        "updatedAt": "2018-05-12 03:46:10",
      },
    ],
  },
}
`;
